config_example.conf

```diff
@@ -5,6 +5,9 @@ admins = <admin,ids>
 [db]
 path=db.sqlite
 
+[core]
+peer_active_time=<hours>
```

Comment:
perhaps specify a default amount of hours instead of a placeholder?

Reply:
this is an example, no need to specify some amount

Reply:
examples are usually modified *as needed*, that's the reason you wrote `path=db.sqlite` and not `path=<db_path>`